packages/shared/lib/typings/activity.ts

```diff
@@ -19,3 +21,24 @@ export enum ActivityAsyncStatus {
     Claimed = 'claimed',
     Expired = 'expired',
 }
+
+export enum ActivityDirection {
+    In = 'in',
+    Out = 'out',
+}
+
+export interface IActivity {
+    id: string
+    type: string
+    timestamp: string
+    activityType: ActivityType
+    direction: ActivityDirection
+    confirmed: boolean
+    internal: boolean
+    amount: string
+    subjectAccountName: string
+    subjectAddress: string
+    token: ITokenMetadata
+    conversion?: number
+    fiatAmount?: number
+}
```

Comment:
We should move this code to the `core/wallet` module that's where the other activity-based logic is.

Reply:
You mean the whole file?

Reply:
We are currently sticking to one file per enum and one file per interface, in the respective folders.

Reply:
So lets move all of these into there own file in wallet/interface or wallet/enum